start

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env python3
```

Comment:
This is a really neat trick. Note that the file still is called simply `./start`. Users don't need to do anything differently, as long as they have `python3` installed on their machine. 

This is part of why I called the script `./start` and not `./start.sh`. The other reason is less typing hah. Although if we did call it `./start.sh`, we could do this trick! File extensions can be a lie - which is important to know for security!